apps/server/src/services/rundown-service/rundownCache.ts

```diff
@@ -76,7 +78,7 @@ export async function init(initialRundown: Readonly<OntimeRundown>, customFields
  * @param rundown
  */
 export function generate(
-  initialRundown: OntimeRundown = persistedRundown,
+  initialRundown: DatabaseOntimeRundown | OntimeRundown = persistedRundown,
```

Comment:
suggestion: `OntimeRundown` is structurally compatible to `DatabaseOntimeRundown`, so we dont need both. just adding the smaller one (`DatabaseOntimeRundown`) is sufficient